shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/sqlserver/dcl/SQLServerRevokeStatement.java

```diff
@@ -17,13 +17,21 @@
 
 package org.apache.shardingsphere.sql.parser.sql.dialect.statement.sqlserver.dcl;
 
+import lombok.Getter;
 import lombok.ToString;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.column.ColumnSegment;
 import org.apache.shardingsphere.sql.parser.sql.common.statement.dcl.RevokeStatement;
 import org.apache.shardingsphere.sql.parser.sql.dialect.statement.sqlserver.SQLServerStatement;
 
+import java.util.LinkedList;
+import java.util.List;
+
 /**
  * SQLServer revoke statement.
  */
 @ToString
+@Getter
 public final class SQLServerRevokeStatement extends RevokeStatement implements SQLServerStatement {
+    
+    private final List<ColumnSegment> columns = new LinkedList<>();
```

Comment:
@ThanoshanMV Can we use Collection<ColumnSegment> here?

Reply:
Hi @strongduanmu, I've updated it.

Reply:
> Hi @strongduanmu, I've updated it.

Thank you @ThanoshanMV, but I found travis ci build error, can you check it on your machine?

Reply:
@strongduanmu I tried to search and have come up with a possible reason! Please let me know if this is the real reason behind ;) We're changing a list of `ColumnSegment` from `List<ColumnSegment>` to `Collection<ColumnSegment>` in the `SQLServerRevokeStatement` as `((CollectionValue<ColumnSegment>) visit(each)).getValue()`'s compile-time type (at line 141 in `SQLServerDCLStatementSQLVisitor.java`) will be `Collection<ColumnSegment>`. The `getColumns()`'s compile-time type will be `Collection<ColumnSegment>`.

So basically, we're getting a `Collection<ColumnSegment>` and adding a `Collection<ColumnSegment>` into it at compile-time. To keep consistency among the datatype at compile-time, we're changing like that. If this is not, please let me know the actual reason :)